doc/RFCs/username_specifications.md

```diff
@@ -0,0 +1,84 @@
+#### _Username_ specifications for Caliopen accounts
+
+This document describes valid **username** for the creation of an account within Caliopen instances.
+
+#### Preambule
+Living in the XXI century, Caliopen's users should be able to take (almost) any username they want to. The following specifications are as open as possible.
+
+##### Definition :
+An **username** is the unique identifier an user makes use of to create an account within a Caliopen instance. 
+Username is only an identifier for the user's account : it is not necessarily the user's real name, or email, or nickname… It will only be used as a credential for the purpose of identifying the user when logging in Caliopen. The username is unique within a Caliopen instance.
+
+NB : once an user has chosen an _username_, she/he will be able to create or add some « identities », that are made of : first name, family name, email, etc.
+
+##### Format :
+ 
+* Username are made of utf-8 "characters".  
+By "character" we mean a single Unicode grapheme, encoded as a single Unicode code point.
+* Username **is** case sensitive.
```

Comment:
the username must definitly be case insensitive to avoid identity scam/squatting and so